bookstore-front/src/core/Card.js

```diff
@@ -13,8 +13,8 @@ const Card = ({
     cartUpdate = false,
     showRemoveProductButton = false,
     showFullDescription = false,
-    setRun,
-    run
+    setRun = f => f,
+    run = undefined
```

Comment:
Why are you putting this back in? You are always passing down values from `Cart` for `setRun` and `run`, you shouldn't need to give them default values.

Reply:
done